src/ui/osx/TogglDesktop/Feature/TimeEntryEditor/AutoComplete/Input/AutoCompleteTextField.swift

```diff
@@ -188,6 +188,19 @@ class AutoCompleteTextField: UndoTextField, NSTextFieldDelegate, AutoCompleteVie
     func didTapOnCreateButton() {
         autoCompleteDelegate?.autoCompleteDidTapOnCreateButton(self)
     }
+
+    override func performKeyEquivalent(with event: NSEvent) -> Bool {
+        // Prevent beep sound on return key
+        if let key = Key(rawValue: Int(event.keyCode)) {
```

Comment:
Look like I have to find better solution to prevent a beep sound. 

Basically, if I return true if the key is tab (like what I did for the Enter key), it leads to weird behavior when I could't tab to move to next view in some controls (even it isn't a AutoComplete TextField). I investigated it, and it turns out that whenever the EditorController receives the key, it will pass this event to all sub-views. If we return true, it will skip the action.